config/services.go

```diff
@@ -22,6 +23,9 @@ import (
 	"github.com/rs/zerolog"
 )
 
+// callbackURLPattern is a regex pattern to check the callback URL (host)
+var callbackURLPattern = regexp.MustCompile(`^https?://`)
```

Comment:
Additionally we should check if it's not a `localhost`.

ChatGPT suggests sth like this:
```golang
regexp.MustCompile(`^https?://(?!localhost|127\.0\.0\.1)([^:/\s]+)`)
```

Reply:
This one is harder, because GO compiler throws an error:
`panic: regexp: Compile(`^https?://(?!localhost|127\.0\.0\.1)([^:/\s]+)`): error parsing regexp: invalid or unsupported Perl syntax: `(?!``
when we try to use **?!** in regexp.
---
What I can do is to check for localhost additionally, for example with:
`go
if strings.Contains(appConfig.Nodes.Callback.Host, "localhost") {
    return nil, errors.New("invalid callback host: cannot contain 'localhost'")
}
`